examples/gin/go.mod

```diff
@@ -1,13 +0,0 @@
-module github.com/airbrake/gobrake/v5/examples/gin
-
-go 1.15
-
-replace github.com/airbrake/gobrake/v5 => ../..
```

Comment:
We no longer have these `replace` directives. As result, when I follow your README instructions, I get:

```
% go run main.go
# command-line-arguments
./main.go:55:10: undefined: "github.com/airbrake/gobrake/v5/gin".New
```

Because the example tries to use the release version (which hasn't been released yet). Why did we change the way this example works? (source code version vs release version)

Reply:
We will merge the change and the example in the same merge, so once we release the new version, everything will be in sync(example and the New() function).

Reply:
Ok, cool. Why do we not use the source code version of gobrake anymore?

Reply:
We won't be including mod and sum files in the example application. this will come from the user environment. But we can use replace keyword when we are developing the gobrake package in local environment.